pylibs/otns/cli/OTNS.py

```diff
@@ -463,7 +471,6 @@ def get_masterkey(self, nodeid: int) -> str:
         Get masterkey.
 
         :param nodeid: target node ID
-        :return:
```

Comment:
This function seems returning something...

Reply:
I will add `-> None` to all methods that return `None` in another PR.